src/components/table/table.stories.js

```diff
@@ -158,7 +223,73 @@ storiesOf('Table', module)
 
       return (
         <State store={ store } parseState={ state => ({ ...state, children: buildRows(pageSize, totalRecords) }) }>
+          <Table
+            actionToolbarChildren={ (context) => {
+              return [
+                <Button disabled={ context.disabled } key='single-action'>
+                    Test Action
+                </Button>,
+                <MultiActionButton
+                  text='Actions' disabled={ context.disabled }
+                  key='multi-actions'
+                >
+                  <Button>foo</Button>
+                  <Button>bar</Button>
+                  <Button>qux</Button>
+                </MultiActionButton>
+              ];
+            } }
+            path='/countries'
+            caption={ caption }
+            shrink={ shrink }
+            highlightable={ highlightable }
+            pageSize={ pageSize }
+            selectable={ selectable }
+            paginate={ paginate }
+            actions={ { delete: { icon: 'bin' }, settings: { icon: 'settings' } } }
+            totalRecords={ totalRecords }
+            showPageSizeSelection={ showPageSizeSelection }
+            onChange={ handleChange }
+            theme={ theme }
+            size={ size }
+            isZebra={ isZebra }
+          />
 
+        </State>
+      );
+    },
+    {
+      info: { text: info },
+      notes: { markdown: notes }
+    },
+  )
+  .add(
+    'with inputs',
+    () => {
+      const pageSize = text('pageSize', '5');
+      const selectable = boolean('selectable', false);
+      const highlightable = boolean('highlightable', false);
+      const shrink = boolean('shrink', false);
+      const caption = text('Caption', 'Country and Country Codes');
+      const totalRecords = text('totalRecords', '50');
+      const paginate = boolean('paginate', false);
+      const showPageSizeSelection = paginate && boolean('showPageSizeSelection', false);
+      const theme = select('theme', OptionsHelper.tableThemes, Table.defaultProps.theme);
+      const size = select('size', OptionsHelper.tableSizes, Table.defaultProps.size);
+      const isZebra = boolean('zebra striping', false);
+      const inputType = select(
```

Comment:
common knobs could be extracted to a separate object and then reused with the rest operator